testdata/project-v3-with-deploy-image/go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	k8s.io/api v0.25.0
 	k8s.io/apimachinery v0.25.0
 	k8s.io/client-go v0.25.0
+	k8s.io/klog/v2 v2.70.1
```

Comment:
Cons: It will add a new dep for the projects which increases the cost for maintainability.